website/docs/r/spring_cloud_service.html.markdown

```diff
@@ -75,6 +77,20 @@ The following arguments are supported:
 
 The `config_server_git_setting` block supports the following:
 
+* `service_runtime_subnet_id` - (Required) Specifies the subnet id to host Azure Spring Cloud Service Runtime.
+
+* `app_subnet_id` - (Required) Specifies the subnet id to host Azure Spring Cloud Apps.
+
+* `cidr` - (Required) Specifies the Spring Cloud service reserved CIDR.
```

Comment:
what does this mean in practice?

Reply:
"(Required) Comma-separated list of IP address ranges in CIDR format. The IP ranges are reserved to host underlying Azure Spring Cloud infrastructure, which should be 3 at least /16 unused IP ranges, must not overlap with any Subnet IP ranges."
How do you think?

Reply:
so then we should also add MinItems: 3 to the schema - @njuCZ can you update that? Perhaps this wants to be:

> (Required) A list of (at least 3) CIDR ranges (at least `/16`) which are used to host the Spring Cloud infrastructure, which must not overlap with any Subnet IP ranges.